### ISSUE TITLE
the 0.2.2.0 release is broken.

### ISSUE THREAD

**Issue by phadej** [swamp-agr/blaze-textual#15]
I made a revision blacklisting it.

```
Configuring library for blaze-textual-0.2.2.0..
Preprocessing library for blaze-textual-0.2.2.0..
Building library for blaze-textual-0.2.2.0..

Blaze/Text/Int.hs:27:0: error:
     error: #elif with no expression
     #elif
     ^
   |
27 | #elif
   | ^
 ```

See https://matrix.hackage.haskell.org/#/package/blaze-textual/0.2.2.0/ghc-8.10.1@1630306313

**Comment by phadej** [comment]
I suspect that you have used some modern (and more lenient) CPP. I'd rather write

```haskell
#if !(MIN_VERSION_base(4,15,0))
import GHC.Num (quotRemInteger)
#endif
```

**Comment by ocharles** [comment]
Note that it's only broken on < 9. This does build on GHC 9

**Comment by ocharles** [comment]
Ah, looks like this is fixed in 0.2.2.1 so I think this can be closed now!
